### DISCUSSION TITLE
Does anyone got mov-cli work on Windows 8.x era?

### DISCUSSION THREAD

**Discussion by Andrew2592009** [mov-cli/mov-cli#341] [General]
Well i tried to install it on my host machine (OS: Windows 8.1) and it did install successfully and configure everything right until i tried to play a video and it complain about an error inside a plugin (I used the otaku-watcher plugin). Has anyone actually got it to work on Windows 8(.1)? (I'm not good at English so sorry if it's a bit confusing)

**Comment by ananasmoe** [comment]
It should work on 8.x, the plugin is just broken it seems.

**Reply by Andrew2592009** [reply]
May i see how you configure the config.toml file? I doubt it would help but may i see it?

**Reply by Andrew2592009** [reply]
If the plugin is broken, then is there a workaround for this?

**Reply by THEGOLDENPRO** [reply]
> If the plugin is broken, then is there a workaround for this?

No there isn't, you'll have to wait until it's fixed by the plugin authors. Maybe try updating your plugins now, it's probably already fixed.
